docs/quick-start/next-steps.txt

```diff
@@ -0,0 +1,35 @@
+.. _laravel-quick-start-next-steps:
+
+==========
+Next Steps
+==========
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: learn more
+
+Congratulations on completing the quick start tutorial!
+
+After you complete these steps, you have a Laravel web application that
+uses the {+odm-long+} to connect to your MongoDB deployment, run a query on
+the sample data, and render a retrieved result.
+
+In this tutorial, you created a Laravel web application that uses the
+{+odm-long+} to connect to your MongoDB deployment, render the result
+of a query, and write data to a MongoDB deployment hosted on
+MongoDB Atlas.
```

Comment:
I: these two paragraphs pretty much repeat the same information, so I don't think you need both

Reply:
Good catch, I forgot to delete one!